### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.4.7][IntelliJ][Wipro]: Keyboard focus is not visible on "Help" button in Run\debug configurations window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1439]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select "edit configurations" present at right side of the dialog.
4. "Run/Debug Configurations" dialog will open.
5. Tab to Help (?) button and hit Space bar and observe keyboard focus .

**Expected Result:**
Keyboard focus is should visible on "Help" button in Run\debug configurations window.

**Actual Result:**
Keyboard focus is not visible on "Help" button in Run\debug configurations window.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![no keyboard focus in edit confog for help button](https://user-images.githubusercontent.com/22956130/35728027-0ef9fede-07bf-11e8-9993-92a619d74954.png)

**Comment by Eskibear** [comment]
duplicated with https://github.com/Microsoft/azure-tools-for-java/issues/1456

**Comment by jdneo** [comment]
* **Importance of experience**: minimal. Very low usage for the help action.
* **Quality of experience**: not impacted. Users can view guidance in the [docs.microsoft.com](https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable#deploy-your-web-app-to-azure). 
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749901
